examples/src/main/java/io/kubernetes/client/examples/WebSocketsExample.java

```diff
@@ -40,19 +40,25 @@ public static void main(String... args) throws ApiException, IOException {
         "GET",
         client,
         new WebSockets.SocketListener() {
-          public void open(String protocol, WebSocket socket) {}
+          private volatile WebSocket socket;
 
-          public void close() {
-            // Trigger shutdown of the dispatcher's executor so this process can exit cleanly.
```

Comment:
Why did we remove this shutdown code?

Reply:
it's the interface changed so we don't need this method anymore